sentry/src/main/java/io/sentry/SentryEnvelope.java

```diff
@@ -53,27 +53,14 @@ public SentryEnvelope(
     this.items = items;
   }
 
-  public static @NotNull SentryEnvelope fromSession(
+  public static @NotNull SentryEnvelope from(
       final @NotNull ISerializer serializer,
-      final @NotNull Session session,
+      final @NotNull Object obj,
       final @Nullable SdkVersion sdkVersion)
       throws IOException {
     Objects.requireNonNull(serializer, "Serializer is required.");
-    Objects.requireNonNull(session, "session is required.");
+    Objects.requireNonNull(obj, "obj is required.");
 
-    return new SentryEnvelope(
-        null, sdkVersion, SentryEnvelopeItem.fromSession(serializer, session));
-  }
-
-  public static @NotNull SentryEnvelope fromEvent(
-      final @NotNull ISerializer serializer,
-      final @NotNull SentryEvent event,
-      final @Nullable SdkVersion sdkVersion)
-      throws IOException {
-    Objects.requireNonNull(serializer, "Serializer is required.");
-    Objects.requireNonNull(event, "Event is required.");
-
-    return new SentryEnvelope(
-        event.getEventId(), sdkVersion, SentryEnvelopeItem.fromEvent(serializer, event));
+    return new SentryEnvelope(null, sdkVersion, SentryEnvelopeItem.from(serializer, obj));
```

Comment:
`fromEvent` was passing `event.getEventId()` instead of `null` in the 1st param, would this cause a bug?

Reply:
Good catch! I did not seem to cause any issues and events were reported correctly, but I guess in some edge cases it could fail.

Reply:
I added overloaded version of this method.